client/lib/interpolate-components/Makefile

```diff
@@ -1,7 +1,23 @@
 REPORTER ?= spec
-MOCHA ?= ../../../node_modules/.bin/mocha
+
+ifneq ("$(wildcard node_modules/.bin/)","")
+NODE_BIN ?= ./node_modules/.bin
+else
+NODE_BIN ?= ../../../node_modules/.bin
+endif
+
+BABEL ?= $(NODE_BIN)/babel
+MOCHA ?= $(NODE_BIN)/mocha
 
 test:
 	@NODE_ENV=test NODE_PATH=test:../../../client $(MOCHA) --compilers jsx:babel/register --reporter $(REPORTER)
```

Comment:
Should we keep `client` in the path? Shouldn't we make sure the module is independent from the rest of Calypso?